src/Common/Acl/AclMain.php

```diff
@@ -225,6 +225,32 @@ public static function aclCheckCore($section, $value, $user = '', $return_value
             }
         }
 
+        $remoteAddr = $_SERVER['REMOTE_ADDR'] ?? null;
```

Comment:
Note that $_SERVER['REMOTE_ADDR'] does not always get what you are looking for. This is why the log engine uses following function to collect info for this type of thing:
https://github.com/openemr/openemr/blob/61e4f5bc1d0d5c12661ad2d46b1d398426d138ca/library/sanitize.inc.php#L20
And then uses this info to place following into the log:
```
$ip = collectIpAddresses();
$ip['ip_string']
```